DTS-STN/canadian-dental-care-plan
# AB#3017 -- Refactoring `cct-service` and `interop-service` into `letters-service`

### Description
Since the application will go through Interop to get the letters (which in turn will call CCT), there's no need to name a service module after an implementation detail of a depending API. This PR combines all service operations related to letters into a new `letters-service`.

More refactoring will be done in a future PR (ie. removing CCT base url environment variables, more unit tests).
I removed the "unit" tests for `cct-service` as they appear to be more like integration tests.

### Related Azure Boards Work Items
[AB#3017](https://dev.azure.com/DTS-STN/Canada%20Dental%20Care%20Plan/_workitems/edit/3017)

### Checklist

### Test Instructions
Run the application and navigate to `/en/letters`. Verify that we can still see letters.
